corehq/apps/ota/utils.py

```diff
@@ -212,4 +213,4 @@ def update_device_id(user, device_id):
         if not user.is_demo_user:
             updated = user.update_device_id_last_used(device_id)
             if updated:
-                user.save(fire_signals=False)
+                user.save(fire_signals=toggles.ENIKSHAY.enabled(user.domain))
```

Comment:
@esoergel I think you did this original PR. Is there any reason to not use `settings.SERVER_ENVIRONMENT == 'enikshay'` That would save a couch/redis request everywhere

Reply:
That seems reasonable - I think we could also make this call the `device_id` update thing directly and not handle the other user signals.  Lemme take a crack at that.